__tests__/moment_spec.re

```diff
@@ -62,18 +62,9 @@ let () = {
         test
           "instantiation with format"
           (
-            fun () =>
-              Just (
-                Equal
-                  true
-                  (
-                    Moment.isSame
-                      (moment format::"YYYY-MM-DD" "2017-04-01")
-                      (moment format::"YYYY-MM-DD" "2017-04-01")
-                  )
-              )
+            fun () => Just (Equal true (Moment.isSame (moment "2017-04-01") (moment "2017-04-01")))
           );
-        test ".now" (fun () => Just (Equal true (moment_now () |> Moment.isValid)));
+        test ".now" (fun () => Just (Equal true (momentNow () |> Moment.isValid)));
```

Comment:
I kkind of get what you're trying to do from a JS perspective, but I don't really think it works very well without function overloading. And it's inconsistent with the convention used by the rest of the BuckleScript ecosystem. See [Js.Date](https://bloomberg.github.io/bucklescript/api/Js.Date.html) for example